hoodie-client/src/main/java/com/uber/hoodie/table/HoodieTable.java

```diff
@@ -136,11 +136,18 @@ public TableFileSystemView getCompletedFileSystemView() {
     return new HoodieTableFileSystemView(metaClient, metaClient.getCommitsTimeline());
   }
 
+  /**
+   * Get only the completed (no-inflights) commit + deltacommit timeline
+   */
+  public HoodieTimeline getCompletedCommitsTimeline() {
+    return metaClient.getCommitsTimeline().filterCompletedInstants();
+  }
+
   /**
    * Get only the completed (no-inflights) commit timeline
    */
   public HoodieTimeline getCompletedCommitTimeline() {
-    return metaClient.getCommitsTimeline().filterCompletedInstants();
+    return metaClient.getCommitTimeline().filterCompletedInstants();
```

Comment:
guess cleaner is still going to call this and you just changed other usages?

Reply:
Yes, exactly.